spec/features/course/level_management_spec.rb

```diff
@@ -17,32 +17,38 @@
     context 'As a Course Administrator' do
       before do
         login_as(user, scope: :user)
+        visit course_levels_path(course)
       end
 
       scenario 'I can view course levels' do
-        visit course_levels_path(course)
-
         (1..3).each do |i|
           expect(page.find('tr', text: i * 100)).to have_selector('td', text: i)
         end
       end
 
       scenario 'I can create a course level' do
-        visit course_levels_path(course)
         find_link(nil, href: new_course_level_path(course)).click
-        fill_in 'level_experience_points_threshold', with: 100
+        fill_in 'level_experience_points_threshold', with: 400
 
-        expect do
-          click_button I18n.t('helpers.submit.level.create')
-        end.to change(course.levels, :count).by(1)
+        expect { click_button I18n.t('helpers.submit.level.create') }.
+          to change(course.levels, :count).by(1)
       end
 
       scenario 'I can delete a course level' do
-        visit course_levels_path(course)
+        expect { find_link(nil, href: course_level_path(course, levels[0])).click }.
+          to change(course.levels, :count).by(-1)
+      end
+
+      context 'for the default level' do
```

Comment:
Okay. Since the deleting of default level should be handled at the ability, then there is no need for a feature spec. 

In any case, for feature specs I assume that I should just write out the entire story instead of having context?